docs/development/getting-started-locally.md

```diff
@@ -23,24 +25,96 @@ In addition to than an echo server will be deployed in its own Namespace which c
 
-1. Deploy an auditing policy.
+```bash
+export KUBECONFIG=$(pwd)/../gardener/example/gardener-local/kind/multi-zone/kubeconfig
+```
+
+### Creating the auditing `Extension.operator.gardener.cloud` resource:
+
+```bash
+make extension-operator-up
+```
+
+The corresponding make target will build the auditing admission and extension container images, OCI artifacts for the admission runtime and application charts, and the extension chart. Then, the container images and the OCI artifacts are pushed into the default skaffold registry (i.e. `garden.local.gardener.cloud:5001`). Next, the auditing `Extension.operator.gardener.cloud` resource is deployed into the KinD cluster. Based on this resource the gardener-operator will deploy the auditing admission component, as well as the auditing ControllerDeployment and ControllerRegistration resources.
+
+### Creating a Shoot Cluster
+
+1. Target the Garden cluster.
+
+```bash
+export KUBECONFIG=$(pwd)/../gardener/dev-setup/kubeconfigs/virtual-garden/kubeconfig
+```
+
+2. Create a secret containing the credentials used for mTLS.
+
+```bash
+kubectl -n garden-local create secret generic echo-server-creds \
+    --from-file=ca.crt=example/local-setup/dev/certs/ca.crt \
+    --from-file=client.crt=example/local-setup/dev/certs/client.crt \
+    --from-file=client.key=example/local-setup/dev/certs/client.key
+```
+
+3. Deploy an auditing policy.
 
 [`example/local-setup/policy.yaml`](../../example/local-setup/policy.yaml) contains a Policy specification:
 ```bash
-kubectl create -f example/local-setup/policy.yaml
+kubectl apply -f example/local-setup/policy.yaml
 ```
 
-2. Create a Shoot cluster.
+4. Create a Shoot cluster.
 
 [`example/local-setup/shoot.yaml`](../../example/local-setup/shoot.yaml) contains a Shoot specification with the `auditing` extension:
 ```bash
-kubectl create -f example/local-setup/shoot.yaml
+kubectl apply -f example/local-setup/shoot.yaml
 ```
 
-3. Once the Shoot namespace is created in the seed cluster create a NetworkPolicy which will allow traffic from the auditlog forwarder to the echo server.
+5. Once the Shoot namespace is created in the seed cluster create a NetworkPolicy which will allow traffic from the auditlog forwarder to the echo server.
 [`example/local-setup/netpol.yaml`](../../example/local-setup/netpol.yaml) contains a NetworkPolicy allowing communication between the auditlog forwarder and the echo server:
 ```bash
-kubectl create -f example/local-setup/netpol.yaml
+kubectl apply -f example/local-setup/netpol.yaml
 ```
+
+### Delete the auditing `Extension.operator.gardener.cloud` resource
+
+Make sure the environment variable `KUBECONFIG` points to the operator's local KinD cluster and then run:
+```bash
+make extension-operator-down
```

Comment:
Before removing the extension, it must not be used, i.e. the shoot has to be deleted or reconfigured to not use the extension.